Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/Snapshots/ComplexCalculator_Add_Generated.g.cs

```diff
@@ -4,44 +4,63 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.IO;
+using Microsoft.Extensions.DependencyInjection;
 using Amazon.Lambda.Core;
+using System.Text.Json;
 
 namespace TestServerlessApp
 {
     public class ComplexCalculator_Add_Generated
     {
-        private readonly ComplexCalculator complexCalculator;
-        private readonly Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer serializer;
+        private readonly ServiceProvider serviceProvider;
 
         public ComplexCalculator_Add_Generated()
         {
             SetExecutionEnvironment();
-            complexCalculator = new ComplexCalculator();
-            serializer = new Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer();
+            var services = new ServiceCollection();
+
+            // By default, Lambda function class is added to the service container using the singleton lifetime
+            // To use a different lifetime, specify the lifetime in Startup.ConfigureServices(IServiceCollection) method.
+            services.AddSingleton<ComplexCalculator>();
+            services.AddSingleton<Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer>();
+
+            var startup = new TestServerlessApp.Startup();
+            startup.ConfigureServices(services);
+            serviceProvider = services.BuildServiceProvider();
```

Comment:
The TestServerlessApp contains a `StartUp` class and this is why the function uses DI. I am not sure why did it not generate these parts in our existing snapshots

Reply:
Nevermind, this was because we never included the `Startup.cs` file in the initial sources for the `ComplexCalculator` snapshot test - https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/SourceGeneratorTests.cs#L217

We do that for the `SimpleCalculator` snapshot test - https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/SourceGeneratorTests.cs#L139